src/conponents/Info.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGem } from '@fortawesome/free-solid-svg-icons';
+import {
+  faCuttlefish, faHtml5, faCss3Alt, faSquareJs, faPython, faReact, faGitAlt,
+} from '@fortawesome/free-brands-svg-icons';
+
+const InfoCard = () => (
+  <div className="information">
+    <div className="infoCard">
+      <div className="about">
+        <h3 className="sub--title">
+          About
+        </h3>
+        <p className="description">
+          I&apos;m a Full-Stack Software Developer located in Nigeria.
+          I have great passion for UI creation, effect,
+          animation and dynamic user experiences.
+          <br />
+          <br />
+          I try to keep up with security and best practices,
+          and am always looking for new things to learn.
+        </p>
+      </div>
+      <div className="interest">
+        <h3 className="sub--title">
+          Interest
+        </h3>
+        <p className="description">
+          I&apos;m interested in working on ambitious projects with
+          positive people and goal oriented companies.
+          <br />
+          <br />
+          I am a Liverpool FC fan. I love swimming,
+          watching TV series and travelling.
+        </p>
+      </div>
+    </div>
+    <div className="stack">
+      <h3 className="sub--title stack-title">
+        Stack
+      </h3>
+      <div className="stack-container">
+        <div className="stack-wrap">
+          <FontAwesomeIcon
+            className="html stack-icon"
+            icon={faHtml5}
+          />
+          <span className="stack-name">
+            HTML5
+          </span>
+        </div>
+        <div className="stack-wrap">
+          <FontAwesomeIcon
+            className="css stack-icon"
+            icon={faCss3Alt}
+          />
+          <span className="stack-name">
+            CSS3
+          </span>
+        </div>
+        <div className="stack-wrap">
+          <FontAwesomeIcon
+            className="javascript stack-icon"
+            icon={faSquareJs}
+          />
+          <span className="stack-name">
```

Comment:
Please use the map function here to render the dynamic data with the same pattern.

Reply:
Sure! I wasn't aware that the JavaScript map method can be used in JSX at the time of writing this project. I just made a research about that and implemented it in a `hotfix` branch.